src/trace-plugin-interface.js

```diff
@@ -111,9 +111,10 @@ var nullSpan = {
  * plugin themselves
  * TODO(kjin): Should be called something else
  */
-function PluginAPI(agent) {
+function PluginAPI(agent, pluginName) {
   this.agent_ = agent;
   this.logger_ = agent.logger;
+  this.pluginName_ = pluginName;
```

Comment:
Up to you. If you include the logic here, make sure the test changes from https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/392/ are also reflected.